db.go

```diff
@@ -70,6 +70,13 @@ type DB struct {
 	// re-sync during recovery.
 	NoFreelistSync bool
 
+	// IsFreelistTypeArray sets the backend freelist type to array which is simple but endures
+	// dramatic performance degradation if database is large and framentation in freelist is common.
+	// The alternative one is using segregated freelist, it is faster in almost all circumstances
+	// but it doesn't guarantee that it offers the smallest page id available. In normal case it is safe.
+	// if you depend on this(it offers the smallest page id available), you may set this to true.
+	IsFreelistTypeArray bool
```

Comment:
can we make this an option rather than a boolean typed selector?

Reply:
the old options are boolean-like(like NoFreelistSync)... I think it is ok to keep this way